app/components/views/AccountsPage/AccountRow/Row.js

```diff
@@ -0,0 +1,61 @@
+import { FormattedMessage as T } from "react-intl";
+import { Balance, VerticalAccordion } from "shared";
+import "style/Fonts.less";
+import "style/AccountRow.less";
+
+const isImported = ({ accountName }) => accountName === "imported";
```

Comment:
It's better to check whether it's the imported account by account number (since that's entirely fixed) rather than name.